client/layout/guided-tours/config-elements.js

```diff
@@ -188,7 +218,8 @@ export class Quit extends Component {
 	}
 
 	onClick = () => {
-		this.context.quit();
+		const { tour, tourVersion, step, isLastStep } = this.context;
+		this.context.next( { tour, tourVersion, step, isLastStep } );
```

Comment:
This says `.next` instead of `.quit`. Is this a typo? On my workdir I changed this to `.quit`, otherwise the tour restarts as soon as it's done.